examples/MLIRLinalg/makefile

```diff
@@ -136,6 +136,76 @@ linalg-matmul-optimize-run:
 	${MLIR_CPU_RUNNER} ${OPT_FLAG} -e main -entry-point-result=void \
 		-shared-libs=${MLIR_RUNNER_UTILS} -shared-libs=${MLIR_C_RUNNER_UTILS}
 
+linalg-batch-matmul-optimize-run:
+	@${BUDDY_OPT} linalg-matmul.mlir ${MLIR_OPT_OPTIONS} \
+		-batchmatmul-optimize="step-placeholder=64" \
```

Comment:
It is recommanded to use `vector-size` instead of  `step-placeholder` since it is easyer to understand.